evalml/pipelines/components/transformers/transformer.py

```diff
@@ -68,3 +68,6 @@ def fit_transform(self, X, y=None):
             except MethodPropertyNotFoundError as e:
                 raise e
         return _convert_to_woodwork_structure(X_t)
+
+    def _get_feature_provenance(self):
```

Comment:
Keeping this private for now since it's only used for permutation importance but this mechanism can help with https://github.com/alteryx/evalml/issues/1347